_posts/2021-06-10-part 3.md

```diff
@@ -14,3 +14,6 @@ date:  2021-06-10
 
 
 Today was pretty short. I only worked three hours today!. Since I'm an TeenWorker, I only can work 24 hours a week. So no overtime or things like that. Today, I got back to the WorkStation setup. It was a little easier than yesterday but I still asked Madison some questions. Lastly, I learned some basic commands for Git Bash. I went over a tutorial Madison sent me. She told to watch it and learn as many of the commands as I could. Some I knew and some I didn't. I'm not finished yet because I had to started my blog! I promised myself I would find how to end of my posts and I think I found something I like! Well I only have an hour left. So Anthony out! Do you like it?? I kinda do and to be honest, that's the best it's going to get haha. 
+
+![WorkStation Setup](Images/Part 3 image.jpg)
+![Git Commands](Images/Part 3 (2) .jpg)
```

Comment:
Will you change your folder and image names? It would be best if the image folder was lowercase. (Image -> image). It would also be good if you could lowercase and dasherize the names, as well as changing the names to make them more descriptive. For example, instead of Images/Part 3 (2) .jpg, something like... images/pull-request.jpg. You can look at this for folder for more examples. http://github.kub.org/web/web.github.kub.org/tree/master/images